priv/repo/migrations/20220511095327_create_keyword_uploads.exs

```diff
@@ -0,0 +1,15 @@
+defmodule GoogleSearchDataViewer.Repo.Migrations.CreateKeywordUploads do
+  use Ecto.Migration
+
+  def change do
+    create table(:keyword_uploads) do
+      add :name, :string
```

Comment:
For the `:name` and `:status` fields, i think we can set it as `null: false` because these fields always need the data.